src/fairchem/data/oc/utils/geometry.py

```diff
@@ -0,0 +1,243 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+if TYPE_CHECKING:
+    from ase.cell import Cell
+
+# Code adapted from https://github.com/henriasv/molecular-builder/tree/master
+
+
+class Geometry(ABC):
+    """
+    Base class for geometries
+    """
+
+    @abstractmethod
+    def __init__(self):
+        pass
```

Comment:
can probably get rid of the __init__ altogether, since `packmol_structure` is abstract and the ABC has not attributes